distribution/src/main/resources/docker-compose/docker-compose.yml

```diff
@@ -178,6 +178,7 @@ services:
       APP_CONFIG_PLUGIN_PROCESS_SERVICE: false
       APP_CONFIG_PROVIDER: ECM
       APP_CONFIG_AUTH_TYPE: BASIC
+      APP_CONFIG_PLUGIN_KNOWLEDGE_RETRIEVAL: false
```

Comment:
I think, that by default you can enable KR feature, as this project is all about integration with HxI

Reply:
thing is that when project is running with mocked endpoints and we enable KR it will be available on ADW UI but it will not response - BE mocks do not cover FE responses

Reply:
Oh! Maybe we should add those mocks? 🤔

Reply:
It's a good idea. Maybe we can do that in separate PR

Reply:
I'm fine with that